core-bundle/tests/Command/MigrateCommandTest.php

```diff
@@ -452,7 +452,7 @@ public function testAbortsOnWrongServerVersion(string $format): void
             $this->assertSame('problem', $json['type']);
             $this->assertSame($errorMessage, $json['message']);
         } else {
-            $this->assertSame('[ERROR] '.$errorMessage, trim(preg_replace('/\s*\n\s*/', ' ', $display)));
+            $this->assertSame('[ERROR] '.$errorMessage, trim(preg_replace('/\s*\n\s*/', "\n", $display)));
```

Comment:
This needs to be reverted to fix the test.

Reply:
the JSONLD tests will fail then, because they contain a line break in the message 😕

Reply:
See https://github.com/contao/contao/pull/5169/files#r946929127